scripts/native-pack-tool/source/platforms/openharmony.ts

```diff
@@ -34,20 +34,35 @@ export class OpenHarmonyPackTool extends NativePackTool {
         await this.generateCMakeConfig();
 
         const ohosProjDir = this.paths.platformTemplateDirInPrj;
-        const cocosXRoot = ps.normalize(Paths.nativeRoot);
         const platformParams = this.params.platformParams;
         const assetsDir = this.paths.buildAssetsDir;
         // local.properties
         await cchelper.replaceInFile([
             { reg: '^sdk\\.dir.*', text: `sdk.dir=${platformParams.sdkPath}` },
         ], ps.join(ohosProjDir, 'local.properties'));
 
+        const abisList = (platformParams.appABIs && platformParams.appABIs.length > 0) ? platformParams.appABIs : ['armeabi-v7a'];
+        
         // entry/build-profile.json5
+        const buildCfgFile = ps.join(ohosProjDir, 'entry/build-profile.json5');
         await cchelper.replaceInFile([
             { reg: '={DRES_DIR}', text: `=${cchelper.fixPath(this.paths.buildDir!)}` },
             { reg: '={DCOMMON_DIR}', text: `=${cchelper.fixPath(process.env.COMMON_DIR || '')}` },
             { reg: /"compileSdkVersion": *,/, text: `"compileSdkVersion": ${platformParams.apiLevel},}` },
-        ], ps.join(ohosProjDir, 'entry/build-profile.json5'));
+        ], buildCfgFile);
+
+        //write abi
+        try {
+            let buildCfgContent = fs.readFileSync(buildCfgFile);
+            let buildCfgJson = JSON5.parse(buildCfgContent.toString());
+            buildCfgJson.buildOption.externalNativeOptions.abiFilters = abisList;
```

Comment:
- can calculate abiFilters here
- and the variable name is better to `abiFilters`